lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View Teams in the Renewable Energy Topic</span><i class="icon fa fa-arrow-right"></i></a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card square-card topic-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Topic</p>
+              <h3 class="card-title">Solar Power Cells</h3>
+              <p class="card-description">Discuss the current technology trajectory of solar power cells</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper has-actions">
+            <div class="card-meta-details">
+              <p class="meta-detail team-count">34 Teams</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View  Teams in the Solar Power Cells Topic</span><i class="icon fa fa-arrow-right"></i></a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card square-card topic-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Topic</p>
+              <h3 class="card-title">Wind Energy</h3>
+              <p class="card-description">Explore how we can harness wind energy, either through large turbine farms, mid-ocean wind farms, and other types of wind energy that would be cut off from this listing because it is so long</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper has-actions">
+            <div class="card-meta-details">
+              <p class="meta-detail team-count">34 Teams</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View Teams in the Wind Energy Topic</span><i class="icon fa fa-arrow-right"></i></a>
+            </div>
+          </div>
+        </div>
+
+        <hr />
+        <br />
+
+        <div class="card list-card team-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Team</p>
+              <h3 class="card-title">Wind Energy</h3>
+              <p class="card-description">Explore how we can harness wind energy, either through large turbine farms, mid-ocean wind farms, and other types of wind energy that would be cut off from this listing because it is so long</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper">
+            <div class="card-meta-details">
+              <div class="meta-detail team-members">
+                <span class="member-count">12 / 15 Members</span>
+                <span class="member-thumbs">
+                  <img alt="AlexeiK" href="" />
```

Comment:
until they create a hybrid img-link (not a bad idea...) you can't be puttin href's on an img ;)
Can we make these an inline ```<ul>``` ?